python/ray/_raylet.pyx

```diff
@@ -678,6 +685,11 @@ cdef CRayStatus check_signals() nogil:
     return CRayStatus.OK()
 
 
+cdef void exit_handler() nogil:
+    with gil:
+        sys.exit(0)
```

Comment:
I don't think this should be hard-coded 0, would be best to propagate it but if anything it should at least be nonzero

Reply:
I considered this, but the plumbing is nontrivial and there isn't a use case for the exit code (we certainly don't check it).